src/ploomber/spec/taskspec.py

```diff
@@ -299,6 +299,7 @@ def to_task(self, dag):
 
         if 'grid' in data:
             data_source_ = data["source"]
+            grid_number_suffix = data.pop('grid_number_suffix', False)
```

Comment:
the default should be True, so we dont break existing pipelines. please change the default, update the tests (if anything breaks) and update the documentation